utils/api_info_v2.py

```diff
@@ -60,4 +61,8 @@
     # Go!
     r = requests.get(api_info_url, auth=auth)
 
-    print(r.text)
+    #print(r.text)
+
+    # For printing unicode instead of unicode escape sequences
```

Comment:
Is the output supposed to be JSON or human readable? `pprint` will be just rude to the user, since they can't use the raw JSON in another script; `pprint` does python-repr compatible output (generally).

It's better to implement `--raw` like the upload api to do `print(r.text)` and otherwise format the output into something like
```
#12345: 'Some Torrent Name' uploaded by 'Username' (123Mb)
  Anime / English-translated - (Complete, Trusted)
```
etc etc.
But, you know, on second thought maybe I'll do all that myself.